docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -3,15 +3,38 @@
 
 [role="_abstract"]
 
-A devfile describes and defines a cloud native development environment.
+Devfiles are YAML files that describe and define developer workspaces or cloud-native development environments. Devfiles also describe best practices for end-to-end application development while providing a way for being customized or extended to fit a team's current development practices. Instead of forcing a new workflow or tools, devfiles provide instructions for automating what was previously done manually.
 
-Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple cloud native resources. Any number of identical workspaces can be created from the same devfile. They can be shared in various ways including a devfile registry or together with the source code of a project.
+Devfiles are consumed by tools that transform the definition into a cloud workspace composed of multiple containers and cloud-native resources. Also, any number of identical workspaces can be created from the same devfile. This way, workspaces can be used by different teams and shared in various ways, including a devfile registry or together with the source code of a project.
 
+A single devfile can include:
+
+* Guidance about runtime images
+* Examnple code
+* Build and CI commands
+* Deployment options
+
+The benefits of devfiles include:
+
+* Reducing the gap between development and deployment
+* Having a registry for finding and hosting devfiles
+* Producing consistent build and run behaviors
+* Allowing easy devfile customization
+
+.Who can use devfiles?
+
+Stack providers use devfiles to develop their runtime stacks once and then use these runtime stacks across different tools. Therefore, with devfiles, stack providers do not need to build and maintain custom development tools.
+
+Tool providers use devfiles to build runtime support for each runtime. Now, the devfile runtime teams build and run applications on their servers, giving tool providers more time.
+
+Developers use devfiles to focus only on developing their applications. With devfiles, developers do not need to set up the environment for building and running their applications. Instead, developers have the flexibility to use different tools, like Che and odo.
 
-[role="_additional-resources"]
 .Additional resources
 
-* xref:api-reference.adoc[]
-* xref:authoring-stacks.adoc[]
-* xref:migrating-to-devfile-v2.adoc[]
-* xref:devfile-samples.adoc[]
+[role="_additional-resources"]
+
+* To see an example tool that uses devfile support, go to https://developers.redhat.com/products/odo/overview[odo].
+* To learn devfile terminologies, go to xref:api-reference.adoc[]
+* To learn what you can do to a devfile, go to xref:authoring-stacks.adoc[]
+* To update your devfile, go to xref:migrating-to-devfile-v2.adoc[]
+* To learn more about devfiles, go to xref:devfile-samples.adoc[]
```

Comment:
It's highly project-specific: in several projects I worked with the common word preceding links was "see" (instead of "go to"), but I've never heard it was a rule or company style. Feel free to ignore this if you use "go to" consistently in the docs.

In several trainings and meetings writers discussed xrefs and the general consensus was "Don't use xrefs" :) However, if you never experienced issues while building docs, I don't see why you shouldn't use them. I guess writers ran into troubles while working on some legacy docs.